aws/resource_aws_lambda_event_source_mapping.go

```diff
@@ -61,6 +61,13 @@ func resourceAwsLambdaEventSourceMapping() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.IsRFC3339Time,
 			},
+			"topics": {
```

Comment:
I kept this as `topics` to match the API even though (at the moment) that can be only one topic specified.

Reply:
maybe this should be `topic` as there would be multiple blocks?

Reply:
Because it's a set of simple types (strings) then the syntax is `topics = ["blah"]`, so no multiple blocks.